tests/main/services-stop-mode-sigkill/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: "Check that refresh-modes sigkill works"
 
+details: |
+    Check that a snap service can be configured to be stopped with the sigterm
```

Comment:
Technically, those are not signals but a means to use signals to stop the service.

Reply:
I've rephrased this now. Thank you